slides/02-data-budget.qmd

```diff
@@ -24,59 +24,68 @@ knitr:
 
 ##  {background-image="https://media.giphy.com/media/Lr3UeH9tYu3qJtsSUg/giphy.gif" background-size="40%"}
 
-## Data on tree frog hatching
+## Data on Chicago taxi trips
 
-![](images/Hatching-process.jpg)
+![](images/taxi.png)
 
-## Data on tree frog hatching
+::: footer
+Credit: <https://unsplash.com/photos/7_r85l4eht8>
+:::
+
+## Data on Chicago taxi trips
 
--   Red-eyed tree frog embryos can hatch earlier than their normal \~7 days if they detect potential predator threat!
--   Type `?stacks::tree_frogs` to learn more about this dataset, including references.
--   We are using a slightly modified version from stacks.
+-   The city of Chicago releases anonymized trip-level data on taxi trips in the city.
+-   We pulled a sample of 10,000 rides occurring in early 2022.
+-   Type `?modeldatatoo::data_taxi()` to learn more about this dataset, including references.
+-   We are using a slightly modified version from the modeldatatoo data.
 
 ```{r}
 library(tidymodels)
+library(modeldatatoo)
 
-data("tree_frogs", package = "stacks")
-tree_frogs <- tree_frogs %>%
-  mutate(t_o_d = factor(t_o_d),
-         age = age / 86400) %>%
-  filter(!is.na(latency)) %>%
-  select(-c(clutch, hatched))
+taxi <- data_taxi() %>%
+  mutate(month = factor(month, levels = c("Jan", "Feb", "Mar", "Apr"))) %>% 
```

Comment:
are things going to break if we exclude this line?

I kinda like this a a good example of what a factor is capable of.

Reply:
I don't think they will break - I included it so that a logistic regression doesn't try to estimate coefficients for the empty factor levels. If we ask learners to fit that model, we should address the factor levels somehow but it doesn't have to be in this way.